CPPs/Jogador.cpp

```diff
@@ -0,0 +1,38 @@
+#include "../HPPs/Jogador.hpp"
+#include <iostream>
+
+Jogador::Jogador(int ID, string nome, vector<Monstrinho *> equipe)
+    : Treinador(ID, nome, equipe) {}
+
+void Jogador::imprimeEquipe()
+{
+    for (int i = 0; i < equipe.size(); i++)
+    {
+        cout << equipe[i]->getNome() << " vida: " << equipe[i]->getHP() << endl;
```

Comment:
Coloca o index do monstrinho na frente também pra ficar claro na hora de selecionar ele nas vezes que precisar, imagino que quando o carioca for fazer a função de usar um item ele deve usar esse metodo